### ISSUE TITLE
Poster Error

### ISSUE THREAD

**Issue by Yasharah** [indico/indico#3038]
i get this error when i try to print the poster:

```
ERROR DETAILS
--
 
Exception type: | <type 'exceptions.TypeError'>
Exception message: | _draw_item() takes exactly 7 arguments (6 given)
Traceback: | File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/legacy/webinterface/rh/base.py", line 493, in process     profile_name, res = self._do_process(profile)    File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/legacy/webinterface/rh/base.py", line 464, in _do_process     res = self._process()    File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/events/management/controllers/posters.py", line 77, in _process     return send_file('Poster-{}.pdf'.format(self.event_new.id), pdf.get_pdf(), 'application/pdf')    File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/designer/pdf.py", line 89, in get_pdf     self._build_pdf(canvas)    File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/events/posters.py", line 63, in _build_pdf     self._draw_item(canvas, item, text, config.margin_horizontal, config.margin_vertical)
Request handler: | RHPrintEventPoster
URL: | https://indico.age.mpg.de/event/10/manage/print-poster/5/92c6ffd3-a49d-4d0a-a150-d7c503c22cdb
Params: | confId = 10uuid = 92c6ffd3-a49d-4d0a-a150-d7c503c22cdbtemplate_id = 5
HTTP headers: | Cookie: _biz_uid=2411c802ca334fca8a526c8737ace637; _biz_nA=5; _biz_pendingA=%5B%5D; _ga=GA1.2.1246023865.1501238743; _gid=GA1.2.350738334.1501574457; indico_session=*** User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36 Connection: keep-alive Referer: https://indico.age.mpg.de/event/10/manage/designer/ Dnt: 1 Host: indico.age.mpg.de Upgrade-Insecure-Requests: 1 Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8 Accept-Language: en-US,en;q=0.8 Accept-Encoding: gzip, deflate, br
Logged user: | MEISTER, Jacqueline <jmeister@age.mpg.de>
```

**Comment by pferreir** [comment]
I see @ThiefMaster redacted out the `indico_session` token. Please be careful pasting error reports here, they may contain sensitive information and allow others to impersonate accounts.
And please try to paste the tracebacks in a readable format.

**Comment by Yasharah** [comment]
Traceback: | File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/legacy/webinterface/rh/base.py", line 493, in process     profile_name, res = self._do_process(profile)    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/legacy/webinterface/rh/base.py", line 464, in _do_process     res = self._process()    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/events/management/controllers/posters.py", line 77, in _process     return send_file('Poster-{}.pdf'.format(self.event_new.id), pdf.get_pdf(), 'application/pdf')    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/designer/pdf.py", line 89, in get_pdf     self._build_pdf(canvas)    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/events/posters.py", line 63, in _build_pdf     self._draw_item(canvas, item, text, config.margin_horizontal, config.margin_vertical)

**Comment by ThiefMaster** [comment]
The exception message is missing from your traceback...

**Comment by Yasharah** [comment]
Exception type: | <type 'exceptions.TypeError'>
-- | --
Exception message: | _draw_item() takes exactly 7 arguments (6 given)

Traceback: | File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/legacy/webinterface/rh/base.py", line 493, in process     profile_name, res = self._do_process(profile)    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/legacy/webinterface/rh/base.py", line 464, in _do_process     res = self._process()    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/events/management/controllers/posters.py", line 77, in _process     return send_file('Poster-{}.pdf'.format(self.event_new.id), pdf.get_pdf(), 'application/pdf')    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/designer/pdf.py", line 89, in get_pdf     self._build_pdf(canvas)    

File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/modules/events/posters.py", line 63, in _build_pdf     self._draw_item(canvas, item, text, config.margin_horizontal, config.margin_vertical)

**Comment by ThiefMaster** [comment]
I think this patch fixes the bug - we'll include it in an updated 1.9.11 release soon: c9ead7ca10094e140a1eee8d64a5cefb0986347c

**Comment by ThiefMaster** [comment]
I just released v1.9.11.dev15 which fixed this
